packages/terra-menu/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 6.86.0 - (March 1, 2024)
+
+* Changed
+  * Minor dependency version bump.
+
 ## 6.85.0 - (February 28, 2024)
 
 * Fixed
```

Comment:
Can you move `Fixed sub-menu height issue when it exceeds viewport size.` under March 1st

Reply:
You can remove the minor dependency version bump.

Reply:
Updated - https://github.com/cerner/terra-framework/pull/2059/commits/8f389e309f1ec8f1a179c9bc5cce3d8c9e6208c0